dev-util/hindent/hindent-5.3.3.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.7.9999
```

Comment:
I recommend using the latest version of hackport (`hackport update && hackport merge hindent`) as opposed to copying the ebuild, as the dependencies _may_ have changed. Also new versions of hackport are "aware" of additional things such as Hackage revisions which will be added to the ebuild it generates.

Reply:
Thanks! I'll remember next time ;)